WorldBank-Transport/DRIVER
# Include all language options in date-utils

Overview
-----

This PR makes all of the languages that the `gregorian` calendar object knows about in regional
options have corresponding entries in our `languageMap` in date-utils. It also makes it so that if
there's a language we don't know about, we just use a default calendar and format.

Testing
-----

- add some new languages to your language options
- reprovision app
- verify that you don't get console errors when using calendars for any of them

[finishes #154032496]
